src/mono/mono/component/mini-wasm-debugger.c

```diff
@@ -1,3 +1,4 @@
+#if defined(HOST_WASM) && !defined(HOST_WASI)
```

Comment:
This doesn't look quite right:

* Below, on line 23, there's a check for `#ifdef HOST_WASM` which doesn't make sense if we already know that's true on line 1. There's also an `#else` block after that which would never take effect so presumably could be removed.
* The final part of this file, from lines 482-490, used to take effect in the `HOST_WASI` case, but they no longer do because of the change on line 1. As a result, there's now a [compilation error](https://github.com/SteveSandersonMS/dotnet-wasi-sdk/runs/7693985370?check_suite_focus=true#step:8:3083) about `mini_wasm_debugger_add_function_pointers` being an undefined symbol.

Reply:
Also now `error : undefined symbol: wasm_debugger_log`

Reply:
Maybe - https://gist.github.com/radical/dbdab755d93dc3656b9dced9205e21d4 . I don't get any build errors at least.